scripts/fastlane/Fastfile

```diff
@@ -0,0 +1,165 @@
+RELEASE_TAG_PREFIX = 'release_v'
+default_platform(:android)
+
+platform :android do
+  desc "Create a pre-release version by pushing a tag to GitHub, and publishing to Maven"
+  
+  gradle_properties_path = File.expand_path("../../../gradle.properties", __FILE__)
+  gradle_project_root = File.dirname(gradle_properties_path)
+  version_regex = /(\d*)\.(\d*)\.(\d*)(?:-unstable\.)?(\d*)?/
+
+  lane :unstable do
+    latest_tag = last_git_tag(pattern: "#{RELEASE_TAG_PREFIX}*")
+
+    UI.message("Last version #{latest_tag}")
+    next_version = calculate_next_canary_version(prerelease_identifier: 'unstable', 
+                                                release_tag_prefix: RELEASE_TAG_PREFIX,
+                                                from_tag: latest_tag )
+    UI.message("Next version: #{next_version}")
+
+    # Update version information in gradle.properties
+    increment_versions(version: next_version)
+
+    gradle(
+      project_dir: gradle_project_root,
+      task: "assemble",
```

Comment:
Should this be `build`, or `clean build`?  `assemble` doesn't run unit tests, which `build` does.
```suggestion
      task: "build",
```